pyproject.toml

```diff
@@ -25,6 +25,7 @@ classifiers = [
 python = "^3.8.1"
 validator-collection = "^1.5.0"
 requests = "^2.28.2"
+jsonschema = "^3.2.0"
```

Comment:
I'm a little concerned with how locked-down these dependencies are. Are we saying we don't want to allow requests v3? Or validator-collection v2? I think we need to be less restrictive with library versions.

Reply:
it seems fine to me, this works with the latest version of sceptre.  also the latest version of sceptre has a `jsonschema>=3.2,<3.3` dependency.  https://github.com/Sceptre/sceptre/blob/master/requirements/prod.txt#L8

Could you please suggest exactly what you would prefer in this case @jfalkenstein ?

Reply:
Interesting. `"^3.2.0"` means the same as `>=3.2,<3.3` so it's probably fine then.